src/App.tsx

```diff
@@ -62,22 +64,31 @@ function App(): JSX.Element {
 
   return (
     <div className="App">
-      <h1>bee game</h1>
-      <p>
-        Honey: {honey} <br />
-        <button onClick={incrementHoney}>buzz buzz buzz</button>
-      </p>
-      <p>
-        bees: {bees} <br />
-        <button disabled={honey < costOfNextBee} onClick={incrementBees}>
-          gain a bee!
-        </button>
-        <br />
-        cost of next bee: {costOfNextBee} <br />
-      </p>
-      <p>
-        <button onClick={reset}>reset</button>
-      </p>
+      <div className="row">
+        <h1>Bee Game</h1>
+        <div className="column left">
+          <p> Honey: {honey} </p>
+          <Button
+            textToDisplay="buzz buzz buzz"
+            clickFunction={incrementHoney}
+          />
+          <p> </p>
+        </div>
+        <div className="column middle">
+          <p> </p>
+          <Button textToDisplay="reset" clickFunction={reset} />
+          <p> </p>
```

Comment:
The empty paragraphs here are keeping everything vertically aligned. This is fine for now. In the future things like this will probably be their own components with a 'topText' and 'bottomText' prop or something. It's just a little scary to mess with something as ubiquitous as `<p>` at this high of a level.

Reply:
Yeah I plan to that when I also make the top level CSS file for the UI components.